File: 08-pizza-party/PizzaParty.java
class PizzaParty {
  public static void main(String[] args) {
    var line = System.console().readLine("How many people? ");
    var people = Integer.parseInt(line);

    line = System.console().readLine("How many pizzas do you have? ");
    var pizzas = Integer.parseInt(line);

    line = System.console().readLine("How many slices per pizza? ");
    var slices = Integer.parseInt(line);

    var slicesEach = (pizzas * slices) / people;
    var remainder = (pizzas * slices) % people;

    System.out.printf("%n%d people with %d pizza%n", people, pizzas);
    System.out.printf("Each person gets %d piece of pizza.%n", slicesEach);
    System.out.printf("There are %d leftover pieces.%n", remainder);
  }
}

File: 08-pizza-party/pizza_party.py
PIECES_PER_PIZZA = 8

people = int(input("How many people? "))
pizzas = int(input("How many pizzas do you have? "))

total_pieces = pizzas * PIECES_PER_PIZZA
pieces_each = total_pieces // people
leftover_pieces = total_pieces % people

print(f"\n{people} people with {pizzas} pizzas\n"
      f"Each person gets {pieces_each} pieces of pizza.\n"
      f"There are {leftover_pieces} leftover pieces.")

